src/lib/constants/quiz.ts

```diff
@@ -0,0 +1,98 @@
+export interface QuizData {
+  questions: Question[];
+}
+
+interface Question {
+  prompt: string;
+  choices: Choice[];
+}
+
+interface Choice {
+  content: string;
+  // Where color is a valid HTML color
+  color: string;
+  // Where the choice is an index number for use in quiz.svelte
+  // ai => 0, dev => 1, design => 2, algo => 3, N/A => 4
+  match: string;
+}
```

Comment:
We can use an enum to make sure the quiz is being accurately calculated.

```suggestion
export enum TeamMatch {
  TEAMLESS,
  ALGO,
  DEV,
  DESIGN,
  AI,
}

interface Choice {
  content: string;
  // Where color is a valid HTML color
  color: string;
  match: TeamMatch;
}
```

Reply:
Done, reordered enums and gave them unique values so it properly works.

Reply:
LGTM. But...

Note: The reason why I ordered the enum with `TEAMLESS` first is because when left alone, the enum values are 0, 1, 2, 3, and so on. Usually, you want the _missing_ or _unsupported_-like value of your enum to be at the top, as I suggested, because the first value is 0. This is therefore the most consistent placement in the enum. 0 is also used as an initialization value in most languages.